pkg/api/types.go

```diff
@@ -1391,7 +1391,10 @@ const (
 	ResourceMemory ResourceName = "memory"
 	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
 	ResourceStorage ResourceName = "storage"
-	// Number of Pods that may be running on this Node: see ResourcePods
+	// Bandwidth available for egress from a pod (bits/second)
+	ResourceEgressBandwidth = "egress"
```

Comment:
are we missing a "ResourceName " here? (and on line 1397, and in api/v1/types.go)